src/test/scala/swiss/dasch/api/ProjectsEndpointSpec.scala

```diff
@@ -167,6 +171,41 @@ object ProjectsEndpointSpec extends ZIOSpecDefault {
     )
   }
 
+  private val assetOriginalSuite =
+    suite("/projects/<shortcode>/asset/<assetId>/original")(
+      test("given the info file does not exist, it should return Not Found") {
+        val req = Request
+          .get(URL(Path.root / "projects" / "0666" / "assets" / "7l5QJAtPnv5-lLmBPfO7U40" / "original"))
+          .addHeader("Authorization", "Bearer fakeToken")
+        executeRequest(req).map(response => assertTrue(response.status == Status.NotFound))
+      },
+      test("return the original contents") {
```

Comment:
would be nice to have a 3rd test case, for when the API denies access

Reply:
I'm having a problem with overriding the layer. I'll try to figure it out, but you can take a look.

Reply:
You could change the `Mock` implementation to contain a `Ref` to the answer the specific tests desire.

Something like this:
```
class MockString(answer: Ref[String]){

  def getString(): UIO[String] = answer.get

  def setAnswerGetString(str: String) = answer.set(str)

}
```

And in the test you call `mock.setAnswerGetString("foo")` just before calling the method.